src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/TagHelperCompletionProviderTest.cs

```diff
@@ -402,7 +402,34 @@ public void GetCompletionAt_AtAttributeEdge_IndexerAttribute_ReturnsCompletions(
         }
 
         [Fact]
-        public void GetCompletionAt_MinimizedAttributeEdge_ReturnsNoCompletions()
+        public void GetCompletionAt_MinimizedAttributeMiddle_ReturnsCompletions()
+        {
+            // Arrange
+            var service = new TagHelperCompletionProvider(RazorTagHelperCompletionService, HtmlFactsService, TagHelperFactsService);
+            var context = CreateRazorCompletionContext(absoluteIndex: 38 + Environment.NewLine.Length, $"@addTagHelper *, TestAssembly{Environment.NewLine}<test2 boo />", isRazorFile: false, tagHelpers: DefaultTagHelpers);
+
+            // Act
+            var completions = service.GetCompletionItems(context);
+
+            // Assert
+            Assert.Collection(
+                completions,
+                completion =>
+                {
+                    Assert.Equal("bool-val", completion.InsertText);
+                    Assert.Equal(TagHelperCompletionProvider.MinimizedAttributeCommitCharacters, completion.CommitCharacters);
+                    Assert.Equal(CompletionSortTextHelper.HighSortPriority, completion.SortText);
+                },
+                completion =>
+                {
+                    Assert.Equal("int-val", completion.InsertText);
```

Comment:
My knowledge of completion here is lacking. Should this be here? It's not shown because the IDE filters it out, but maybe we're supposed to do the filtering??